src/serverenvironment.h

```diff
@@ -334,6 +334,10 @@ class ServerEnvironment final : public Environment
 	// Find the daylight value at pos with a Depth First Search
 	u8 findSunlight(v3s16 pos) const;
 
+	void updatePos(u16 id, const v3f &pos) {
```

Comment:
```suggestion
	inline void updateObjectPos(u16 id, const v3f pos) {
```

Reply:
:+1: but I don't think the `inline` makes sense for consistency, other inline defs don't have it either